pkg/mapper/iac-providers/cft/config/ecs-cluster.go

```diff
@@ -0,0 +1,127 @@
+/*
+    Copyright (C) 2021 Accurics, Inc.
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+package config
+
+import "github.com/awslabs/goformation/v5/cloudformation/ecs"
+
+// ClusterSettingsBlock holds config for settings attribute
+type ClusterSettingsBlock struct {
+	Name  string `json:"name"`
+	Value string `json:"value"`
+}
+
+// CapacityProviderStrategyBlock holds config for default_capacity_provider_strategy attribute
+type CapacityProviderStrategyBlock struct {
+	Base             int    `json:"base"`
+	CapacityProvider string `json:"capacity_provider"`
+	Weight           int    `json:"weight"`
+}
+
+// LogConfigurationBlock holds config for log_configuration attribute
+type LogConfigurationBlock struct {
+	CloudWatchEncryptionEnabled bool   `json:"cloud_watch_encryption_enabled"`
+	CloudWatchLogGroupName      string `json:"cloud_watch_log_group_name"`
+	S3BucketName                string `json:"s3_bucket_name"`
+	S3EncryptionEnabled         bool   `json:"s3_bucket_encryption_enabled"`
+	S3KeyPrefix                 string `json:"s3_key_prefix"`
+}
+
+// ExecuteCommandConfiguration holds config for execute_command_configuration attribute
+type ExecuteCommandConfiguration struct {
+	KmsKeyId         string                  `json:"kms_key_id"`
+	Logging          string                  `json:"logging"`
+	LogConfiguration []LogConfigurationBlock `json:"log_configuration"`
+}
+
+// ConfigurationBlock holds config for configuration attribute
+type ConfigurationBlock struct {
+	ExecuteCommandConfig []ExecuteCommandConfiguration `json:"execute_command_configuration"`
+}
+
+// EcsClusterConfig holds config for aws_ecs_cluster resource
+type EcsClusterConfig struct {
+	Config
+	ClusterName                     string                          `json:"name"`
+	ClusterSettings                 []ClusterSettingsBlock          `json:"settings"`
+	DefaultCapacityProviderStrategy []CapacityProviderStrategyBlock `json:"default_capacity_provider_strategy"`
+	Configuration                   []ConfigurationBlock            `json:"configuration"`
+}
+
+// GetEcsClusterConfig returns config for aws_ecs_cluster resource
+func GetEcsClusterConfig(e *ecs.Cluster) []AWSResourceConfig {
+
+	clusterSettingsData := make([]ClusterSettingsBlock, len(e.ClusterSettings))
+	for i := 0; i < len(e.ClusterSettings); i++ {
```

Comment:
we can use `range` here to make code cleaner

Reply:
Wouldnt we need i for indexing the clusterSettingsData array ?

Reply:
Try this:
```go
for i := range e.ClusterSettings {
.....
}
```